app/src/main/java/ELME/View/CustomWheelListener.java

```diff
@@ -0,0 +1,33 @@
+package View;
+
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.graphics.ICamera;
+import java.awt.event.MouseWheelEvent;
+import java.awt.event.MouseWheelListener;
+
+/**
+ * This listeners listens for MouseWheelEvent to allow zooming feature
+ *
+ * @author andru
+ */
+public class CustomWheelListener implements MouseWheelListener{
+    
+    private final ICamera cam;
+    private static final float MILTIPLIER = (float)0.5;
+
+    public CustomWheelListener() {
+        super();
+        this.cam = Game.world().camera();

```

Comment:
If camera changes after construction, the zoom functionality would not work. I recommend not storing a reference to a camera all together. Game.world().camera() could be used in method t set zoom.